fluent-plugin-td.gemspec

```diff
@@ -18,7 +18,6 @@ Gem::Specification.new do |gem|
   gem.license       = "Apache-2.0"
 
   gem.add_dependency "fluentd", [">= 0.14.13", "< 2"]
-  gem.add_dependency "td-client", "~> 1.0"
```

Comment:
You do not need to remove this from dependency just for specifying `git`. Even if you leave `gem.add_dependency "td-client"`, you can still use the latest git repo by the same Gemfile. Leaving missing dependency would rather be problematic if it's released as is.

Reply:
It doesn't seems to work 😕

Reply:
because https://github.com/treasure-data/fluent-plugin-td/pull/35/commits/75dfa521235fda14ae96d5f6c0005089964cd449 was not what I meant. By "the same Gemfile" I meant `gem "td-client", git: "https://github.com/treasure-data/td-client-ruby.git"` in Gemfile. (Otherwise how would it find the git repository?)